apps/condo/bin/prepare-review-deploy.js

```diff
@@ -0,0 +1,112 @@
+const path = require('path')
+
+const { GraphQLApp } = require('@open-keystone/app-graphql')
+
+const conf = require('@open-condo/config')
+
+const { OidcClient } = require('@condo/domains/user/utils/serverSchema')
+
+
+let context
+async function getContext () {
+    if (context) return context
+
+    const resolved = path.resolve('./index.js')
+    const { distDir, keystone, apps } = require(resolved)
+    const graphqlIndex = apps.findIndex(app => app instanceof GraphQLApp)
+    await keystone.prepare({ apps: [apps[graphqlIndex]], distDir, dev: true })
+    await keystone.connect()
+    context = await keystone.createContext({ skipAccessControl: true })
+
+    return context
+}
+
+// https://review-custom-name-***.*** -> "review-custom-name-"
+function getReviewPrefix ({ reviewDomain }) {
+    const condoDomain = conf['CONDO_DOMAIN']
+    // controlled reviewDomain
+    // nosemgrep: javascript.lang.security.audit.detect-non-literal-regexp.detect-non-literal-regexp
+    const findPrefixRegex = new RegExp(`https://(review-[a-zA-Z0-9-]+?-)\\w+\\.${reviewDomain.replaceAll('.', '\\.')}`)
```

Comment:
You can call escapeRegexp from lodash, but I guess it's fine for now? Or do we have any complex branching?

Reply:
use `escapeRegexp` instand `replaceAll`